### PR TITLE
HU Lex migration

### PR DESCRIPTION
**Pull request by dee-luo**
- Ukrainian and Russian pre-survey bots for webchat only (no language selector bot necessary)
- Didn't add a slot type file for Ukrainian yes/no about self, because one exists in the generic configs folder
- Added Russian slot types in generic configs folder
- Didn't need to make any TaskRouter changes, as the Survey stuff was all in place (queue, channel, workflow)
- Made Studio changes directly in the console and updated flow.tftpl directly instead of using a template
- Enable lex feature flag in staging service configuration

### CODE REVIEW

**Comment by dee-luo** [comment]
@janorivera I think I did all the steps, feel okay on the Lex files but I was kind of winging it on the main.tf changes and if there's anything else I need to put in the new TG folder structure... 😬 

@GPaoloni tagging you just as a second pair of eyes if you have time and can validate the steps I took!

**Review by janorivera — COMMENTED** [review]
Hi @dee-luo ! I've added some comments!
Also, it seems that I don't have access to the staging account, but related to the taskRouter I would just confirm that the workflow filter is correct:

{
  "task_routing": {
    "filters": [
      {
        "filter_friendly_name": "CaptureChannel",
        "expression": "isChatCaptureControl==true",
        "targets": [
          {
            "queue": "${task_queues.survey}"
          }
        ]
      }
    ]
  }
}

File: twilio-iac/helplines/hu/configs/lex/uk/bots/pre_survey.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
[the remark this answers is not in the document]
Comment (dee-luo) [reply twilio-iac/helplines/hu/configs/lex/uk/bots/pre_survey.json:1]: Oh you're right. I think we can get rid of this one. I don't think we need the one I created. The main difference between the generic one is that they don't want to ask the Gender question, but that's something that is sent in the intents folder right?

[the remark this answers is not in the document]
Comment (dee-luo) [reply twilio-iac/helplines/hu/configs/lex/uk/bots/pre_survey.json]: Deleted this file, will rely on the generic file

File: twilio-iac/helplines/hu/configs/lex/ru/bots/pre_survey.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
```
[the remark this answers is not in the document]
Comment (dee-luo) [reply twilio-iac/helplines/hu/configs/lex/ru/bots/pre_survey.json:1]: Going to keep this here for now if that's okay! But agreed we can probably make this general in the future

File: twilio-iac/kek-vonal-staging/flow.tftpl [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ${
```
[the remark this answers is not in the document]
Comment (dee-luo) [reply twilio-iac/kek-vonal-staging/flow.tftpl:1]: Updated to use variables and updated the main TF file with the appropriate variables!

**Review by janorivera — APPROVED** [review]
lgtm !

**Pull request merged**